File: get-price-emoji.ts
export default function getPriceEmoji(price: number, average: number) {
  if (price === 0) {
    return "🆓";
  }

  if (price < 0) {
    return "💶";
  }

  if (price < average) {
    return "✅";
  }

  return "❌";
}

File: internal/get_price_emoji.go
package internal

func GetPriceEmoji(price, average float64) string {
	switch {
	case price == 0:
		return "🆓"
	case price < 0:
		return "💶"
	case price <= average:
		return "✅"
	default:
		return "❌"
	}
}
